content/docs/introduction/roadmap.md

```diff
@@ -11,13 +11,20 @@ features and current work, see the issue trackers for the various repositories,
 for example, the [Prometheus
 server](https://github.com/prometheus/prometheus/issues).
 
+### New storage engine
+
+A new more efficient storage engine is in development. It will offer reduced
+resource usage, better inverted indexes, and generally allow Prometheus to
+scale further.
+
 ### Long-term storage
 
 Currently Prometheus has support for storing samples on local disk, as well as
-experimental support for writing data into OpenTSDB and InfluxDB. We plan to
-improve long-term storage support, although the details are not determined yet.
-This might include read-back support from OpenTSDB/InfluxDB in Prometheus, or
-support for other long-term storage backends.
+experimental support for sending data into remote systems via a generic mechanism.
```

Comment:
We agreed that it is confusing enough so that we purged the word "generic" from the code base.

It's not overly confusing here, but I'd still avoid the term as it has many connotations.

(But to put things into perspective: This is a nit compared to removing the single one most needed feature from the roadmap without consulting the other core devs.)